src/OrbitGrpcProtos/capture.proto

```diff
@@ -24,14 +24,16 @@ message CaptureOptions {
   message InstrumentedFunction {
     string file_path = 1;
     uint64 file_offset = 2;
-    uint64 absolute_address = 3;
+    uint64 function_id = 3;
 
     enum FunctionType {
       kRegular = 0;
       kTimerStart = 1;
       kTimerStop = 2;
     }
     FunctionType function_type = 4;
+
+    string function_name = 5;
```

Comment:
Why is this sent to the service?

Reply:
this is to save function_name of the timer in the service side when switching to new capture format.

Reply:
Hmmm ok, this doesn't really belong to this commit then I guess. Also I didn't really understand: Where does it need to be saved by the service? Why isn't it enough that the service sends back `function_id`s to the client?

Reply:
agree, this doesn't really belong to this change. removed.